src/main/java/io/confluent/kafkarest/ProducerPool.java

```diff
@@ -216,4 +270,45 @@ public void onCompletion(
         List<RecordMetadataOrException> results
     );
   }
+
+  public static ProducerPool.ProduceRequestCallback defaultProduceRequestCallback(final AsyncResponse asyncResponse, final Logger log) {
```

Comment:
If you want I can revert this change since it's not mandatory but reduce code duplication (this method is used 3 times)